code/weapon/weapons.cpp

```diff
@@ -6346,6 +6348,11 @@ void weapon_hit( object * weapon_obj, object * other_obj, vec3d * hitpos, int qu
 	if (sci->inner_rad != 0.0f || sci->outer_rad != 0.0f)
 	{
 		if(sci->speed > 0.0f) {
+			if (!sci->rot_defined) {
+				sci->rot_angles.p = frand_range(0.0f, 1.99f * PI);
+				sci->rot_angles.b = frand_range(0.0f, 1.99f * PI);
+				sci->rot_angles.h = frand_range(0.0f, 1.99f * PI);
```

Comment:
Please use the `PI2` contant.  (The range will safely exclude the maximum number.)